lib/routes/main_page.dart

```diff
@@ -314,4 +318,7 @@ class _MainPageState extends State<MainPage> with TickerProviderStateMixin {
         return MapEntry(appInfo.noNotesIllustration, "No notes were added yet");
     }
   }
+  Future<void> sync() async {
+    await SyncRoutine().syncNotes();
```

Comment:
may be better to add the SyncRoutine instance to the getit locator as a singleton so you dont have to always reinstance it

Reply:
I agree, i will add it to the locator.